cordova-lib/src/plugman/registry/registry.js

```diff
@@ -236,15 +231,22 @@ function initSettings() {
     module.exports.settings =
     rc('plugman', {
         cache: plugmanCacheDir,
-        registry: 'http://registry.cordova.io',
+        registry: registryURL,
         logstream: fs.createWriteStream(path.resolve(plugmanConfigDir, 'plugman.log')),
         userconfig: path.resolve(plugmanConfigDir, 'config'),
         'cache-min': oneDay
     });
+
+    // if npm is true, use npm registry. 
+    // ~/.plugman/config overides the above config if it exists. 
+    // Need to reset the registry value in settings 
+    if(npm) {
+        settings.registry = 'http://registry.npmjs.org';
```

Comment:
If `~/.plugman/config` exists, it doesn't matter what I passed in above. It gets overwritten by what's in that `config` file. That is why I have to do the check down here. 

I could move the string to a variable. Doesn't really add much though.